content/docs/command-reference/exp/run.md

```diff
@@ -67,11 +78,14 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
 > In addition to the following, `dvc exp run` accepts the options in `dvc repro`
 > except for `--glob`, `--no-commit`, and `--no-run-cache`.
 
-- `-S [<filename>:]<param_name>=<param_value>`,
-  `--set-param [<filename>:]<param_name>=<param_value>` - set the value of
-  existing `dvc params` for this experiment. `filename` can be any valid params
-  file (`params.yaml` by default). This will override the param values coming
-  from the params file.
+- `-S [<filename>:]<override_pattern>`,
+  `--set-param [<filename>:]<override_pattern>` - set the value of existing
+  `dvc params` for this experiment. `filename` can be any valid params file
+  (`params.yaml` by default). This will override the param file before running
+  the experiment.
+
+  Check the [examples](#example-modify-parameters-on-the-fly) for valid values
+  of `<override_pattern>`.
```

Comment:
Sorry, I didn't mean that we should avoid mentioning Hydra. I think it's important to mention that Hydra override syntax is supported as you did before, but I thought it might be helpful to first note how it works without any special Hydra syntax.

Reply:
Before you had written:

> Valid `override_pattern` values are defined in the
  [Hydra Override syntax](https://hydra.cc/docs/advanced/override_grammar/basic/#modifying-the-config-object).

Would you say that's still correct or do you think we need to specify what is and isn't valid?

Reply:
> I tried to link to the `Modifying the Config Object` section. Maybe we just need to specify that `Modifying the Defaults List` is not supported

I think it will be after https://github.com/iterative/dvc/pull/8093, right? It's not even unsupported as much as inapplicable here I think. IMO it's not worth mentioning.